src/sandstorm/app-index/app-index.c++

```diff
@@ -163,7 +164,38 @@ public:
         httpResponse.setMimeType("application/octet-stream");
         httpResponse.initBody().setBytes(outStream.getArray());
 
-        return kj::READY_NOW;
+        if (!req.isSetState() ||
+            !response.getRoot<SubmissionStatus>().isPending()) {
+          return kj::READY_NOW;
+        }
+
+        // Send notification email to app index reviewers.
+        auto appTitle = indexer.getAppTitle(packageId);
+        auto notificationText = kj::str(
+            "An app package is pending review in the app index.\n\n"
+            "https://alpha.sandstorm.io/grain/NujwEZfut8oZoSdcrFzy9p/\n\n"
+            "title: ", appTitle, "\n"
+            "packageId: ", packageIdString(req.getPackageId()), "\n"
+            "requested state: ", req.getSetState().getNewState(), "\n");
+
+        return session.getPublicIdRequest().send()
```

Comment:
If it fails an exception is thrown. The submitter ends up seeing an error. You could argue that we shouldn't display an error to the submitter given that their app was successfully submitted and only the notification email failed but I do not see a better place for the error report to go that we're actually likely to see, and I'd hate for the submitter to falsely believe that their submission will be looked at when in fact we haven't received any notification and so won't be looking.